libc/watcom/syscall/crt0.c

```diff
@@ -16,11 +16,7 @@
 int _argc;              /* with declaration of main() */
 int _8087;              /* when floating point seen */
 
-/* cstart_ is an external reference created by Watcom C, pulls in asm/segments.asm */
-#pragma aux cstart_ "_*" modify [ bx cx dx si di ]
-extern void cstart_(void);
-
-static noreturn void sys_exit(int status);
+extern void sys_exit(int status);
```

Comment:
I guess what you're saying here is that `noreturn` is ignored since the code is inlined and emitted directly as specified, correct?

What about `static`? Is that also useless since the code is inlined?

Reply:
In line code cannot be static, OW compiler accept it but it is not fully correct.
Take into account that #pragma aux is template which is intanced when you use it.
You can have it in header file. I you are interested in such example look into OW sorce code in bld/watcom/h/tinyio.h or int10.h or int33.h all these files contains many inline functions template that you can simply include such header file in your C cohe and use any #pragma aux template and it include copy of inline code into C code.